frontend/src/app/modules/boards/boards-root/boards-root.component.ts

```diff
@@ -7,7 +7,7 @@ import {QueryUpdatedService} from "core-app/modules/boards/board/query-updated/q
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/8d0ecb4b3ed813f91bb58c934f28a2845d090292/frontend/src/app/modules/boards/boards-root/boards-root.component.ts#L7"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>